test/InputSpec.js

```diff
@@ -240,7 +240,7 @@ describe('Input', function () {
       <Input type="text" label="l" wrapperClassName="wrapper" addonAfter="a" hasFeedback={true} help="h"/>
     );
 
-    let node = instance.getDOMNode();
+    let node = React.findDOMNode(instance);
```

Comment:
Maybe these instances of `React.findDOMNode()` should be replaced with `ReactTestUtils.findRenderedDOMComponentWithTag()` to be more consistent with the other tests in this spec?